webapp/src/components/App/Settings/Users/dialog/UserFormDialog/UserForm.tsx

```diff
@@ -119,11 +125,18 @@ function UserForm(props: Props) {
             helperText={errors.password?.message?.toString()}
             {...commonTextFieldProps}
             {...register("password", {
-              required: t("form.field.required") as string,
-              minLength: {
-                value: PASSWORD_MIN_LENGTH,
-                message: t("form.field.minLength", { 0: PASSWORD_MIN_LENGTH }),
-              },
+              validate: (v) => validatePassword(v) || undefined,
+            })}
+          />
+          <TextField
+            label={t("global.confirmPassword")}
+            type="password"
+            spellCheck
+            error={!!errors.confirmPassword}
+            helperText={errors.confirmPassword?.message?.toString()}
+            {...commonTextFieldProps}
+            {...register("confirmPassword", {
+              validate: (v) => validatePassword(v, getValues("password")),
```

Comment:
Check only equlity here, if the first password field is invalid the form will not be send